.github/workflows/integration_tests.yml

```diff
@@ -1095,33 +1095,35 @@ jobs:
               --resourceUri=helm/keptn/templates/ingress.yaml
           done
 
-      - name: Trigger Keptn Control Plane Delivery
-        id: trigger-keptn-control-plane-delivery
-        uses: keptn/gh-action-send-event@main
-        with:
-          keptnApiUrl: ${{ secrets.META_KEPTN_API_URL }}/v1/event
-          keptnApiToken: ${{ secrets.META_KEPTN_API_TOKEN }}
-          event: |
-            {
-              "data": {
-                "message": "",
-                "project": "${{ env.META_KEPTN_KEPTN_PROJECT }}",
-                "result": "",
-                "service": "keptn",
-                "stage": "dev",
-                "status": "",
-                "labels": {
-                  "github-action-run": "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
-                }
-              },
-              "source": "gh",
-              "specversion": "1.0",
-              "type": "sh.keptn.event.dev.delivery.triggered",
-              "shkeptnspecversion": "0.2.1"
-            }
+  #######################################################################
+  # This job triggers service deliveries and wait for them to pass      #
+  #######################################################################
+  trigger_keptn_delivery:
+    name: Trigger Service Delivery
+    needs: deploy_keptn_with_keptn
+    if: github.event_name == 'schedule'
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        include:
+          - SERVICE_NAME: keptn
+          - SERVICE_NAME: jmeter-service
+          - SERVICE_NAME: helm-service
+    env:
+      SERVICE_NAME: ${{ matrix.SERVICE_NAME }}
+      META_KEPTN_KEPTN_PROJECT: keptn
+    steps:
+      - name: Install Keptn CLI for META_KEPTN
+        id: install_keptn_cli
+        run: |
+          curl -sL https://get.keptn.sh | KEPTN_VERSION=${{ env.META_KEPTN_VERSION }} bash
```

Comment:
This step is executed for every service in the matrix, and takes some time.

Can we cache the Keptn CLI (with a cache key being the specified `META_KEPTN_VERSION`), and only download it if it's not available? e.g. 
```
- name: Cache Keptn CLI
  uses: actions/cache@v2
  with:
    path: |
      /usr/local/bin/keptn <-- not sure if that's the right path
    key: ${{ env.META_KEPTN_VERSION }}
```

Reply:
i added the cache step and also added a condition to not fetch the keptn installer script if there was a cache hit ✅